### DISCUSSION TITLE
beIN SPORTS

### DISCUSSION THREAD

**Discussion by hossain11** [iptv-org/community#284] [Q&A] [answered]
Hello, how can I get on this site beIN SPORTS channels in .m3u8 format

**Comment by BellezaEmporium** [comment, marked as answer]
By paying for beIN Sports.

**Reply by hossain11** [reply]
Brother I want it for free and...

**Reply by BellezaEmporium** [reply]
> Brother I want it for free and...

There's nothing we can do on our side, sorry.

**Reply by hossain11** [reply]
Thank you, brother, where can I buy it? Is there an exact site?

**Reply by BellezaEmporium** [reply]
France : https://www.beinsports.com/france/
Middle East : https://www.bein.com/ar/
US + LATAM : https://www.beinsports.com/us/

**Reply by hossain11** [reply]
Thank you very much brother, thank you very much

**Reply by Mohamed-Salah-1** [reply]
I will never pay for such a company who is trading on the poors happiness and sell it for 10% in Europe
